osmoutils/store_helper.go

```diff
@@ -103,3 +104,29 @@ func gatherValuesFromIteratorWithStop[T any](iterator db.Iterator, parseValue fu
 func noStopFn([]byte) bool {
 	return false
 }
+
+// MustGet gets key from store by mutating result
+// Panics on any error.
+func MustGet(store store.KVStore, key []byte, result proto.Message) {
+	b := store.Get(key)
+	if b == nil {
+		panic(fmt.Errorf("getting at key (%v) should not have been nil", key))
+	}
+	if err := proto.Unmarshal(b, result); err != nil {
+		panic(err)
+	}
+}
+
+// MustSetDec sets dec value to store at key. Panics on any error.
+func MustSetDec(store store.KVStore, key []byte, value sdk.Dec) {
+	MustSet(store, key, &sdk.DecProto{
+		Dec: value,
+	})
+}
+
+// MustGetDec gets dec value from store at key. Panics on any error.
+func MustGetDec(store store.KVStore, key []byte) sdk.Dec {
+	result := &sdk.DecProto{}
+	MustGet(store, key, result)
+	return result.Dec
+}
```

Comment:
Do we just not write Dec's as raw state entries anywhere in our codebase?

Reply:
Technically, it is possible to `json.Marshal` `sdk.Dec` but I was aiming to reuse `MustGet` since the error checking seems pretty much identical, as a result, increasing boilerplate.

using a `proto.Message` wrapper for `Dec` allows to reust the `MustGet` and `MustSet`, reducing boilerplate.

Let me know what you think. I can change this in a future PR if you think json marshaling is preferable